echopype/convert/parse_ad2cp.py

```diff
@@ -240,33 +224,21 @@ def parse_raw(self):
         with open(self.source_file, "rb") as f:
             while True:
                 try:
-                    packet = Ad2cpDataPacket(f, self, self.burst_average_data_record_version)
-                    self.previous_packet_data_record_type = packet.data_record_type
-                    if packet.is_burst():
-                        self.burst_packets.append(packet)
-                    elif packet.is_average():
-                        self.average_packets.append(packet)
-                    elif packet.is_echosounder():
-                        self.echosounder_packets.append(packet)
-                    elif packet.is_echosounder_raw():
-                        self.echosounder_raw_packets.append(packet)
-                    elif packet.is_echosounder_raw_transmit():
-                        self.echosounder_raw_transmit_packets.append(packet)
-                    elif packet.is_string():
-                        self.string_packets.append(packet)
+                    packet = Ad2cpDataPacket(f, self)
+                    self.packets.append(packet)
```

Comment:
This is just a comment: the refactoring/consolidation you have done for `Ad2cpDataPacket` is awesome!